test/repository/identity.test.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @jest-environment node
+ */
+
+import { user } from '../factories/user';
+
+import * as userRepository from '@/repositories/user';
+import * as identityRepository from '@/repositories/identity';
+import { AuthProviders } from '@/constants/enum';
+
+describe('Identity Repository', () => {
+  describe('createIdentity', () => {
+    describe('given valid parameter', () => {
+      it('creates identity', async () => {
+        const userId = 1;
+        const userPayload = { ...user, id: userId };
+        await userRepository.createUser(userPayload);
+        const identityPayload = {
+          user: { connect: { id: userId } },
+          provider: AuthProviders.GOOGLE,
+        };
+        const createdIdentity = await identityRepository.createIdentity(identityPayload);
+
+        expect(createdIdentity.userId).toEqual(userId);
+        expect(createdIdentity.provider).toEqual(AuthProviders.GOOGLE);
+      });
+    });
+  });
+
+  describe('find identity', () => {
```

Comment:
```suggestion
  describe('findByProvider', () => {
```

As you are using this format above